ref(javascript): Streamline queue instrumentation page (#10123)
getsentry/sentry-docs: docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation/queues-module.mdx

| Attribute                            | Type   | Description                                                         |
| :----------------------------------- | :----- | :------------------------------------------------------------------ |
| `messaging.message.id `              | string | The message identifier                                              |
| `messaging.destination.name`         | string | The queue or topic name                                             |
| `messaging.message.body.size`        | number | Size of the message body in bytes                                   |
| `messaging.message.retry.count `     | number | The number of times a message was attempted to be processed         |
| `messaging.message.receive.latency ` | number | The time in milliseconds that a message awaited processing in queue |

Use `Sentry.continueTrace` to connect your consumer spans to their associated producer spans, and `setStatus` to mark the trace of your message as success or failed.

<Note>

> Your `queue.process` span must exist as a child within a parent span in order to be recognized as a consumer span. If you don't already have a parent consumer span, you can start a new one using `Sentry.startSpan`.

Reviewer: ```suggestion
Your `queue.process` span must exist as a child within a parent span in order to be recognized as a consumer span. If you don't already have a parent for the consumer span, you can wrap the consumer span with a parent span by using `Sentry.startSpan`.
```
